RadixWallet/SARGON/WrappingRadixEngineToolkit/ExtensionsUsedOnTypes/OnlyPossibleInSwift/DummySargonCodable.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+// MARK: - DummySargonCodable
+public protocol DummySargonCodable: Codable {}
+extension DummySargonCodable {
```

Comment:
Some types from RET are made codable by iOS wallet since our Cache client requires it. When caching is done by Sargon, this can go away. And we really should :)

Reply:
Note that caching with Sargon might be harder, as Android uses an actual database, like CoreData, AFAIK.

Reply:
We can let caching take place in iOS and in Android for now, and just provide a simple interface for it, and let Sargon be able to tell FFI host "load_cached_value_for_key" - and we can let key be a String for now. I think that is feasible.

Reply:
That, or we can have Sargon ask for a given resource by address, and it is up to the Wallet to use the current infra to return either the value from the GW or from cache.

Reply:
yes indeed.